src/app/component/layout/components/footer/footer.component.scss

```diff
@@ -79,6 +78,10 @@ li {
     padding: 0;
   }
 
+  .footer_bottom-part {
+    float: left;
```

Comment:
I would suggest not using "float" property since it's kinda depricated

Reply:
It works fine...I think It does not matter

Reply:
Floats should not be used for layout anymore (though you can still use them for the original purpose - floating text around images).

Flexbox is now widely supported and is better for layout.